source/MaterialXFormat/File.cpp

```diff
@@ -75,7 +77,11 @@ string FilePath::asString(Format format) const
 
     if (format == FormatPosix && isAbsolute())
     {
-        str += "/";
+        // Don't prepend a POSIX separator on a Windows absolute path
+        if (_vec.empty() || !std::isalpha(_vec[0][0]) || (_vec[0][1] != ':'))
```

Comment:
For safety maybe check that _vec[0] is a string of at least 2 characters, before accessing _vec[0][0] and _vec[0][1].

Reply:
Good idea. I'll fix this.